digitalpublicgoods/notesmaster.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "NotesMaster",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://notesmaster.com/termsofuse"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "The content is authored using the Tiny MCE Editor which is a WYSISWYG editor based on HTML. This makes it versatile in the insertion of links and the embedding of other learning software such as H5P, Prezi, Slideshare, nearpod, Wordwall, Vyond, Kazoo, Quizizz, Storyboard That, Pixton. The platform operates across all browsers, is mobile responsive and allows all content to be printed and downloaded to MS Word Format. The Notesmaster YouTube channel guides users in how to register, find classes and generally get started:https://www.youtube.com/channel/UCBFmZoPITZFZl-DFYbx-2nA/videos. This support link is accessible on the website by clicking the ‘?’ at the top."
```

Comment:
This is not a valid URL for the documentation for this project. Please provide a valid documentation URL.

Reply:
@lacabra the link below addresses the above issue raised:

System requirements - https://notesmaster.com/systemrequirements